### PR TITLE
Update contributing documentation to document new website deployment strategy

### PR DESCRIPTION
**Pull request by sbesson**
Follow up of https://github.com/openmicroscopy/www.openmicroscopy.org/pull/239 and https://github.com/openmicroscopy/prod-playbooks/pull/88, this updates the contributing pages to document the new process for releasing a new version of the website.

### CODE REVIEW

File: contributing/jekyll.rst [hunk 1]
```diff
@@ -53,21 +53,33 @@ Editing the websites
...
-review by triggering the
-:jenkinsjob:`WWW-merge` job. The site will be viewable at
-`<https://snoopycrimecop.github.io/www.openmicroscopy.org/>`_.
-
-Once the PR is merged, the live site can currently only be updated by one of
-the sysadmin team running a
-`playbook <https://github.com/openmicroscopy/prod-playbooks/tree/master/www>`_
-to update the deployment.
+Once a PR is open and has passed Travis, it will be integrating in the staging
+integration branch for the OME Website site via the daily :jenkinsjob:`WWW-merge` job. The staging Jekyll website  will be deployed by the `GitHub Pages`_ service at
```
Comment (mtbc) [inline contributing/jekyll.rst]: maybe remove the double space and break the line

File: contributing/jekyll.rst [hunk 2]
```diff
@@ -53,21 +53,33 @@ Editing the websites
...
+https://snoopycrimecop.github.io/www.openmicroscopy.org for review
```
Comment (mtbc) [inline contributing/jekyll.rst]: end with period

**Comment by manics** [comment]
https://github.com/openmicroscopy/ome-documentation/pull/1887/files#diff-9c7106eb597fe922382de0c1b8097d67R12
> These sites can be built and tested by installing a local copy of Jekyll

The Gemfile was removed in https://github.com/openmicroscopy/www.openmicroscopy.org/pull/239 so does this still work or should you use the docker image instead?

**Comment by mtbc** [comment]
In local testing I would remove the Gemfile because I didn't have what it wanted and things seemed to work fine so I'm glad to see the back of it.

**Comment by jburel** [comment]
Merging for 5.4.7 since it describes the new way to deploy the website

**Pull request merged**